src/documents/tutorial-usingStringTemplates/UsingStringTemplates.md

```diff
@@ -0,0 +1,296 @@
+
+The final piece is handled by an [`expander` block](../ExpansionOfComponentOptions.md).
+
+```
+  transformed: {
+      expander: {
+          func: "{that}.parseTemplates"
+      }
+  },
+```
+
+The `expander` block will be replaced with the value returned by our `parseTemplates` invoker, and `logState` is called with the expanded value, which in our case is a map of transformed values.  We will then have sensible output that we can work with.
+
+### Overriding options from a child component
+
+In the next example, we will look at creating a child component that overrides some of our variables in its default configuration.  Here's the updated code:
+
+```
+  // Sample demonstration of fluid.stringTemplate to parse %variable references...
+  "use strict";
+  var fluid = fluid || require("infusion");
+  var gpii  = fluid.registerNamespace("gpii");
+
+  fluid.registerNamespace("gpii.sandbox.variables.base");
+  fluid.defaults("gpii.sandbox.variables.base", {
+      gradeNames: ["autoInit", "fluid.eventedComponent"],
+      terms:      {
+          one: "base one",
+          two: "base two"
+      },
+      templates: {
+          one: "The term named 'one' is set to '%one'.",
+          two: "The term named 'two' is set to '%two'."
+      },
+      listeners: {
+          onCreate: [
+              {
+                  funcName: "gpii.sandbox.variables.base.logState",
+                  args:     ["{that}", {expander: {func: "{that}.parseTemplates"}}]
+              }
+          ]
+      },
+      invokers: {
+          parseTemplates: {
+              funcName: "fluid.transform",
+              args: ["{that}.options.templates","{that}.transformTemplate"]
+          },
+          transformTemplate: {
+              funcName: "fluid.stringTemplate",
+              args: ["{arguments}.0", "{that}.options.terms"]
+          }
+      }
+  });
+
+  gpii.sandbox.variables.base.logState = function (that, parsed) {
+      console.log("\nMy friends call me '" + that.nickName + "'...");
+      console.log("terms -> one: " + that.options.terms.one);
+      console.log("terms -> two: " + that.options.terms.two);
+      console.log("template one: " + that.options.templates.one);
+      console.log("template two: " + that.options.templates.two);
+      console.log("one, parsed : " + parsed.one);
+      console.log("two, parsed : " + parsed.two);
+  };
+
+  fluid.registerNamespace("gpii.sandbox.variables.child");
+  fluid.defaults("gpii.sandbox.variables.child", {
+      gradeNames: ["autoInit", "gpii.sandbox.variables.base"],
+      templates: {
+          one: "The term named one is set to '%one', also, I am a custom template."
+      },
+      terms:      {
```

Comment:
I agree, my whitespace indenting is not particularly valuable here.   I have replaced with colon space squiggly throughout.